app/Observers/ProjectHasUserObserver.php

```diff
@@ -54,6 +59,8 @@ public function created(ProjectHasUser $projectHasUser): void
                     'custodian_id' => $custodianId,
                 ]
             );
+
+            $this->notifyUserChanged($projectHasUser, $organisationId, $custodianId);
```

Comment:
using this out of scope fella. You'll need to declare within global scope and null/valid check before using to get around the test failure.